manual/filtering_pane/README.md

```diff
@@ -96,33 +96,33 @@ Matching items must have acqusition time within provided range. Not specyfing st
 
 ### Modification Time
 
-This component alows to query the items by their modification time. Its visual behaviour is the same as for [Acquisition Time](#acquisition-time). It's filtering behaviour hovewer is a combination of [Modified by](#modified-by) and [Acquisition Time](#acquisition-time) logics.
+This component alows to query the items by their modification time. Its visual behaviour is the same as for [Acquisition Time](#acquisition-time). Its filtering behaviour hovewer is a combination of [Modified by](#modified-by) and [Acquisition Time](#acquisition-time) logics.
 
 #### Filtering behaviour
 
-When `Modification Time` checkbox is enabled it takes part in filtering logic and behaves differently when it's combined with annotable fields and differently when it's not.
+When `Modification Time` checkbox is enabled it takes part in filtering logic and behaves differently when combined with annotable fields and differently when not.
 
-Additionally for the case of querying data by the specific user and specific modification time it's adviced to combine this criterion with `Modified by`.
+Additionally in case of querying data by the specific user and specific modification time it is advised to combine this criterion with `Modified by`.
 
 ##### Filtering without annotable fields combined
 
-When `Modification Time` criterion is not combined with annotable fields it matches the items where provided timerange matches modification time of __any__ item's annotable field. Please note that such query is more expensive for the system and it might take a bit longer to be handled.
+When `Modification Time` criterion is not combined with annotable fields it matches the items where provided timerange matches modification time of item's __any__  annotable field. Please note that such query is more expensive for the system and it might take a bit longer to be handled.
```

Comment:
```suggestion
When `Modification Time` criterion is not combined with annotable fields it matches the items where provided time range matches modification time of __any__ item's annotable field. Please note that such query is more expensive for the system and it might take a bit longer to be handled.
```